--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   scripts/reinforcement_learning/rsl_rl/train_debug.py
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/__pycache__/traverse_terrain_generator_cfg.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/__pycache__/extreme_traverse_terrians.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/extreme_traverse_terrians.py
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/traverse_terrain_generator_cfg.py
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/rewards.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/terminations.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/rewards.py
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/terminations.py
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/rsl_rl/unitree_go2_rough/2025-10-31_16-56-43/
	logs/rsl_rl/unitree_go2_rough/2025-10-31_16-57-22/
	logs/rsl_rl/unitree_go2_rough/2025-10-31_17-10-35/
	logs/rsl_rl/unitree_go2_rough/2025-10-31_17-16-31/
	logs/rsl_rl/unitree_go2_rough/2025-10-31_18-03-24/
	logs/rsl_rl/unitree_go2_rough/2025-10-31_18-03-59/
	outputs/2025-10-31/16-51-40/
	outputs/2025-10-31/16-56-43/
	outputs/2025-10-31/16-57-22/
	outputs/2025-10-31/17-01-53/
	outputs/2025-10-31/17-03-38/
	outputs/2025-10-31/17-03-52/
	outputs/2025-10-31/17-10-35/
	outputs/2025-10-31/17-16-31/
	outputs/2025-10-31/17-55-10/
	outputs/2025-10-31/18-03-24/
	outputs/2025-10-31/18-03-59/
	source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-313.pyc
	source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/rewards.cpython-313.pyc

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/scripts/reinforcement_learning/rsl_rl/train_debug.py b/scripts/reinforcement_learning/rsl_rl/train_debug.py
index 03beb17..8aba792 100644
--- a/scripts/reinforcement_learning/rsl_rl/train_debug.py
+++ b/scripts/reinforcement_learning/rsl_rl/train_debug.py
@@ -29,8 +29,8 @@ sys.path.insert(0, ROOT)
 sys.argv = [
     "train.py",
     "--task=Isaac-Traverse-Walls-Unitree-Go2-v0",
-    "--headless",
-    "--num_envs=1024",
+    # "--headless",
+    "--num_envs=512",
     "--max_iterations=10000",
 ]
 
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/__pycache__/traverse_terrain_generator_cfg.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/__pycache__/traverse_terrain_generator_cfg.cpython-311.pyc
index fc89660..70ead60 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/__pycache__/traverse_terrain_generator_cfg.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/__pycache__/traverse_terrain_generator_cfg.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/__pycache__/extreme_traverse_terrians.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/__pycache__/extreme_traverse_terrians.cpython-311.pyc
index 5260441..8fe5bcb 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/__pycache__/extreme_traverse_terrians.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/__pycache__/extreme_traverse_terrians.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/extreme_traverse_terrians.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/extreme_traverse_terrians.py
index 93a0f1c..7959828 100644
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/extreme_traverse_terrians.py
+++ b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/extreme_traverse_terrians.py
@@ -69,7 +69,7 @@ def two_walls_terrain(
     cfg, 
     num_goals: int,
 ) -> tuple[np.ndarray, np.ndarray, np.ndarray]:
-    """单段双墙走廊地形（风格与 parkour_hurdle_terrain 一致）"""
+    """单段双墙走廊地形"""
 
     # ------------------------ 基本分辨率设置 ------------------------
     hs = cfg.horizontal_scale
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/traverse_terrain_generator_cfg.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/traverse_terrain_generator_cfg.py
index 7c5d5a6..718828a 100644
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/traverse_terrain_generator_cfg.py
+++ b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/traverse_terrain_generator_cfg.py
@@ -20,6 +20,6 @@ class TraverseSubTerrainBaseCfg(HfTerrainBaseCfg):
     
 @configclass
 class TraverseTerrainGeneratorCfg(TerrainGeneratorCfg):
-    num_goals: int = 8 
+    num_goals: int = 3
     terrain_names: list[str] = [] 
     random_difficulty: bool = False 
\ No newline at end of file
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc
index aaa08c2..2510810 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc
index 149d554..35e840b 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/rewards.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/rewards.cpython-311.pyc
index 43a2daf..b1e9860 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/rewards.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/rewards.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/terminations.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/terminations.cpython-311.pyc
index a89c4b4..f21a999 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/terminations.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/terminations.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/rewards.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/rewards.py
index 8b021b2..9554e21 100644
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/rewards.py
+++ b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/rewards.py
@@ -218,4 +218,4 @@ def reward_collision(
 ) -> torch.Tensor:
     contact_sensor: ContactSensor = env.scene.sensors[sensor_cfg.name]
     net_contact_forces = contact_sensor.data.net_forces_w_history[:,0,sensor_cfg.body_ids]
-    return torch.sum(1.*(torch.norm(net_contact_forces, dim=-1) > 0.1), dim=1)
+    return torch.sum(1.*(torch.norm(net_contact_forces, dim=-1) > 0.1), dim=1)
\ No newline at end of file
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/terminations.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/terminations.py
index 0651b48..32cbda2 100644
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/terminations.py
+++ b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/terminations.py
@@ -16,7 +16,7 @@ from typing import TYPE_CHECKING
 
 from isaaclab.assets import Articulation
 from isaaclab.managers import SceneEntityCfg
-from isaaclab.utils.math  import euler_xyz_from_quat, wrap_to_pi
+from isaaclab.utils.math import euler_xyz_from_quat, wrap_to_pi
 from ..mdp import TraverseEvent
  
 if TYPE_CHECKING:
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py
index 3518091..de1085b 100644
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py
+++ b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py
@@ -234,7 +234,7 @@ class EventCfg:
     #     mode="startup",
     # )
     randomize_rigid_body_mass = EventTerm(
-        func= randomize_rigid_body_mass,
+        func=randomize_rigid_body_mass,
         mode="startup",
         params={
             "asset_cfg": SceneEntityCfg("robot", body_names="base"),
@@ -243,7 +243,7 @@ class EventCfg:
             },
     )
     randomize_rigid_body_com = EventTerm(
-        func= events.randomize_rigid_body_com,
+        func=events.randomize_rigid_body_com,
         mode="startup",
         params={
             "asset_cfg": SceneEntityCfg("robot", body_names="base"),
@@ -387,6 +387,7 @@ class RewardsCfg:
     )
 
 
+
 @configclass
 class TerminationsCfg:
     """Termination terms for the MDP."""