--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   scripts/reinforcement_learning/rsl_rl/train_debug.py
	deleted:    source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/agents/cusrl_ppo_cfg.py
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/observations.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/rsl_rl/unitree_go2_rough/2025-10-31_11-45-57/
	outputs/2025-10-31/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/scripts/reinforcement_learning/rsl_rl/train_debug.py b/scripts/reinforcement_learning/rsl_rl/train_debug.py
index ac539e5..03beb17 100644
--- a/scripts/reinforcement_learning/rsl_rl/train_debug.py
+++ b/scripts/reinforcement_learning/rsl_rl/train_debug.py
@@ -29,7 +29,7 @@ sys.path.insert(0, ROOT)
 sys.argv = [
     "train.py",
     "--task=Isaac-Traverse-Walls-Unitree-Go2-v0",
-    # "--headless",
+    "--headless",
     "--num_envs=1024",
     "--max_iterations=10000",
 ]
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/agents/cusrl_ppo_cfg.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/agents/cusrl_ppo_cfg.py
deleted file mode 100644
index aa93a6a..0000000
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/agents/cusrl_ppo_cfg.py
+++ /dev/null
@@ -1,50 +0,0 @@
-# Copyright (c) 2024-2025 Ziqi Fan
-# SPDX-License-Identifier: Apache-2.0
-
-import cusrl
-from cusrl.environment.isaaclab import TrainerCfg
-
-from isaaclab.utils import configclass
-
-
-@configclass
-class UnitreeGo2RoughTrainerCfg(TrainerCfg):
-    max_iterations = 20000
-    save_interval = 100
-    experiment_name = "unitree_go2_rough"
-    agent_factory = cusrl.ActorCritic.Factory(
-        num_steps_per_update=24,
-        actor_factory=cusrl.Actor.Factory(
-            backbone_factory=cusrl.Mlp.Factory(
-                hidden_dims=[512, 256, 128], activation_fn="ELU", ends_with_activation=True
-            ),
-            distribution_factory=cusrl.NormalDist.Factory(),
-        ),
-        critic_factory=cusrl.Value.Factory(
-            backbone_factory=cusrl.Mlp.Factory(
-                hidden_dims=[512, 256, 128], activation_fn="ELU", ends_with_activation=True
-            ),
-        ),
-        optimizer_factory=cusrl.OptimizerFactory("AdamW", defaults={"lr": 1.0e-3}),
-        sampler=cusrl.AutoMiniBatchSampler(num_epochs=5, num_mini_batches=4),
-        hooks=[
-            cusrl.hook.ValueComputation(),
-            cusrl.hook.GeneralizedAdvantageEstimation(gamma=0.99, lamda=0.95),
-            cusrl.hook.AdvantageNormalization(),
-            cusrl.hook.ValueLoss(),
-            cusrl.hook.OnPolicyPreparation(),
-            cusrl.hook.PpoSurrogateLoss(),
-            cusrl.hook.EntropyLoss(weight=0.008),
-            cusrl.hook.GradientClipping(max_grad_norm=1.0),
-            cusrl.hook.OnPolicyStatistics(sampler=cusrl.AutoMiniBatchSampler()),
-            cusrl.hook.AdaptiveLRSchedule(desired_kl_divergence=0.01),
-        ],
-    )
-
-
-@configclass
-class UnitreeGo2FlatTrainerCfg(UnitreeGo2RoughTrainerCfg):
-    def __post_init__(self):
-        super().__post_init__()
-        self.max_iterations = 5000
-        self.experiment_name = "unitree_go2_flat"
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc
index 9776edb..e247578 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/__pycache__/observations.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/observations.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/observations.py
index 1c9dcf7..c2a87a5 100644
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/observations.py
+++ b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/mdp/observations.py
@@ -31,7 +31,7 @@ class ExtremeTraverseObservations(ManagerTermBase):
         super().__init__(cfg, env)
         self.contact_sensor: ContactSensor = env.scene.sensors['contact_forces']
         self.ray_sensor: RayCaster = env.scene.sensors['height_scanner']
-        self.traverse_event: TraverseEvent =  env.traverse_manager.get_term(cfg.params["traverse_name"])
+        self.traverse_event: TraverseEvent = env.traverse_manager.get_term(cfg.params["traverse_name"])
         self.asset: Articulation = env.scene[cfg.params["asset_cfg"].name]
         self.sensor_cfg = cfg.params["sensor_cfg"]
         self.asset_cfg = cfg.params["asset_cfg"]