--- git status ---
On branch main
Your branch is ahead of 'origin/main' by 1 commit.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/config/__pycache__/traverse.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/__pycache__/traverse_env_cfg.cpython-311.pyc
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/flat_env_cfg.py
	modified:   source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/traverse_env_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/rsl_rl/unitree_go2_rough/2025-10-25_21-23-35/
	outputs/2025-10-25/21-23-35/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/config/__pycache__/traverse.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/config/__pycache__/traverse.cpython-311.pyc
index 9e47d2c..c3a6d23 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/config/__pycache__/traverse.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/terrains/extreme_traverse/config/__pycache__/traverse.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/__pycache__/traverse_env_cfg.cpython-311.pyc b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/__pycache__/traverse_env_cfg.cpython-311.pyc
index 1123cfd..527b4eb 100644
Binary files a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/__pycache__/traverse_env_cfg.cpython-311.pyc and b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/__pycache__/traverse_env_cfg.cpython-311.pyc differ
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/flat_env_cfg.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/flat_env_cfg.py
index 47c53d3..91f0a18 100644
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/flat_env_cfg.py
+++ b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/flat_env_cfg.py
@@ -3,7 +3,7 @@
 
 from isaaclab.utils import configclass
 
-from .rough_env_cfg import UnitreeGo2TraverseEnvCfg
+from .traverse_env_cfg import UnitreeGo2TraverseEnvCfg
 
 
 @configclass
diff --git a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/traverse_env_cfg.py b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/traverse_env_cfg.py
index 6c3b7d3..0ee555b 100644
--- a/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/traverse_env_cfg.py
+++ b/source/unitree_gym/unitree_gym/tasks/manager_based/locomotion/velocity/config/quadruped/unitree_go2/traverse_env_cfg.py
@@ -55,8 +55,8 @@ class UnitreeGo2TraverseEnvCfg(LocomotionVelocityGo2EnvCfg):
         # ------------------------------Events------------------------------
         self.events.randomize_reset_base.params = {
             "pose_range": {
-                "x": (-0.5, 0.5),
-                "y": (-0.5, 0.5),
+                "x": (-1.0, 0.0),
+                "y": (-0.0, 0.0),
                 "z": (0.0, 0.2),
                 "roll": (-0.31, 0.31),
                 "pitch": (-0.31, 0.31),